docs/tutorial.md

```diff
@@ -66,10 +76,14 @@ Connect to the virtual machine:
 vagrant ssh
 ```
 
+
+
 From inside the virtual machine we are going to install all requirements.
 Please note that *mysql-server* will require a password for the root user that you'll need later.
 
 ```
+sudo add-apt-repository ppa:fkrull/deadsnakes
```

Comment:
The right fix would be to use python3 instead of python3.4, already done on another PR